Gemfile.lock

```diff
@@ -14,16 +14,17 @@ GIT
 
 GIT
   remote: https://github.com/fastlane/fastlane
-  revision: c579921dcf7a95ea30c4f947e39c66fa64810354
+  revision: 27fd2c57a9519c03393b21191285caacf075f294
   specs:
-    fastlane (2.89.0)
+    fastlane (2.94.0)
       CFPropertyList (>= 2.3, < 4.0.0)
       addressable (>= 2.3, < 3.0.0)
       babosa (>= 1.0.2, < 2.0.0)
       bundler (>= 1.12.0, < 2.0.0)
       colored
       commander-fastlane (>= 4.4.6, < 5.0.0)
       dotenv (>= 2.1.1, < 3.0.0)
+      emoji_regex (~> 0.1)
```

Comment:
@josh- JOOOOOOSSSSH

Reply:
@KrauseFx Feeeeeelix! 😂

Reply:
Ohh, sorry, wrong Josh, haha, I meant to tag @joshdholtz

Reply:
Yeahhhhh, was looking into replacing this but replacing 100% functionality of that gem felt not worth for the amount of work/maintenance on maintaining our own implementation. Emoji is all over the unicode spectrum (some in ranges and some randomly placed that need a one off regex) and using this gem (for now) makes our maintenance easier 😬